package.json

```diff
@@ -64,7 +65,7 @@
     "deploy": "npm version major && npm run build && firebase deploy",
     "lint": "eslint src",
     "pretest": "npm run lint",
-    "test": "react-scripts test",
+    "test": "react-scripts test --env=jsdom --silent",
```

Comment:
`--silent` also prevents any console.logs in tests from outputting, which can make debugging hard. I would suggest removing this and either living with the warnings or finding a way to suppress them.